plugins/scanners/fossid/src/main/kotlin/events/UploadArchiveHandler.kt

```diff
@@ -131,8 +132,13 @@ class UploadArchiveHandler(
     override suspend fun afterCheckScan(scanCode: String) {
         if (config.deleteUploadedArchiveAfterScan) {
             logger.info { "Deleting uploaded archive for scan '$scanCode'." }
-            service.removeUploadedContent(config.user.value, config.apiKey.value, scanCode)
-                .checkResponse("remove previously uploaded content 2", false)
+
+            try {
+                service.removeUploadedContent(config.user.value, config.apiKey.value, scanCode)
+                    .checkResponse("remove previously uploaded content 2", false)
+            } catch (e: SocketTimeoutException) {
+                logger.info { "SocketTimeoutException while deleting uploaded archive: ${e.message}. Continuing." }
```

Comment:
Nits: I wouldn't repeat the exact name of the exception in this user-facing output. Nor would write something after the (potentially) long exception message.

So how about: "Ignoring timeout while deleting uploaded archive: ${e.message}"

Reply:
Should the `$e.message` be dropped, because the rest of the message already has enough info?

Reply:
I believe it's still valuable to know *why* the timeout occurred, which is what the message could help to understand.